src/main/scala/gitbucket/core/api/MergeAPullRequest.scala

```diff
@@ -0,0 +1,23 @@
+package gitbucket.core.api
+
+/**
+ * https://developer.github.com/v3/pulls/#merge-a-pull-request
+ */
+case class MergeAPullRequest(
+  commit_title: Option[String],
+  commit_message: Option[String],
+  /* TODO: Not Implemented
+  sha: Option[String],*/
+  merge_method: Option[String]
+)
+
+case class SuccessToMergePrResponse(
+  sha: String,
+  merged: Boolean,
+  message: String
+)
+
+case class FailToMergePrResponse(
+  documentation_ur: String,
```

Comment:
`documentation_url` ?

Reply:
@oshiro3 Could you take a look at this?

Reply:
yes. I fix this typo.

@SIkebe 
Thanks for finding the typo.